prow/cluster/jobs/istio/istio/istio.istio.master.yaml

```diff
@@ -455,6 +455,7 @@ presubmits:
         testing: test-pool
   - name: e2e-simpleTests-distroless-master
     <<: *job_template
+    always_run: true
```

Comment:
can we add `skip_report: true` until https://github.com/istio/istio/issues/15414 is fixed?

Reply:
Fixed

Reply:
https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_istio/15608/e2e-simpleTests-distroless-master/3